config/grunt/eslint.js

```diff
@@ -1,21 +1,26 @@
+const grunt = require('grunt');
+
 module.exports = {
     config: {
         options: {
             configFile: 'config/eslint/config.json',
+            fix: grunt.option('fix'),
```

Comment:
This is so I could auto-fix lint errors by running `grunt lint --fix`

Reply:
Thanks for adding this, I actually planned to add something similar for quite some time but never got around to it. It is not related to the other stuff, but I like it. :-)